Resources/config/user_context.xml

```diff
@@ -39,5 +40,9 @@
             <argument />
             <argument />
         </service>
+
+        <service id="fos_http_cache.user_context.anonymous_request_matcher" class="FOS\HttpCacheBundle\UserContext\AnonymousRequestMatcher">
+            <argument />
```

Comment:
this leads to the travis failure. i would define the service without argument at all.